### ISSUE TITLE
is_multicore Releases.v5_0_0 is false!

### ISSUE THREAD

**Issue by talex5** [ocurrent/ocaml-version#59]
ocaml-version claims that 5.0.0 isn't multicore:
```
utop # Ocaml_version.(Configure_options.is_multicore Releases.v5_0_0);;
- : bool = false
```
This is causing CI to fail (e.g. https://github.com/mirage/mirage-crypto/issues/169#issuecomment-1416072152).

**Comment by dra27** [comment]
cf. https://github.com/ocurrent/ocaml-version/issues/58#issuecomment-1399270623. There's an open question as to what this function means for 5.x. In the past, it was referring to if there was a multicore _variant_ of OCaml, but OCaml 5.x _is_ multicore.

**Comment by talex5** [comment]
The doc comment says:
>   (** [is_multicore t] is [true] if this version is a multicore-capable release. *)

`Releases.v5_0_0` is surely a multicore-capable release.

**Comment by dra27** [comment]
Yes, indeed - I'm not arguing, it's just that the function isn't actually used that way (even in ocaml-version - see later in the opam package selection) so it's just not an instantly obvious fix (I thought there was a workaround for this in ocaml-ci in progress?)

**Comment by tmcgilchrist** [comment]
There is a fix in progress for this. Antonin was looking into it.
